frontend/src/api/medicalRecordFunctions.js

```diff
@@ -13,7 +13,29 @@ async function fetchAllPatientMedicalRecords(patientId) {
 
   const medicalRecordList = await response.json();
 
-  return await medicalRecordList;
+  return medicalRecordList;
 }
 
-export { fetchAllPatientMedicalRecords };
+async function createMedicalRecord(medicalRecordBody) {
+  //Create the medical record
+  const response = await fetch(
+    `${import.meta.env.VITE_BASE_URL}/medical-history/create-medical-history`,
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(medicalRecordBody),
+    }
+  );
+
+  if (!response.ok) {
+    throw new Error("Failed to create the new medical history registry");
+  }
+
+  const medicalRecord = await response.json();
```

Comment:
We don't need to await here considering response is already after the `await` of fetch?